src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+};
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(todo.userId).then(data => {
+      setUser(data);
+    });
+  });
```

Comment:
The `useEffect` hook is missing a dependency array. This can cause the effect to run on every render, which is not optimal. Add `[todo.userId]` as the dependency array to ensure the effect runs only when `todo.userId` changes.